ros_buildfarm/templates/ci/ci_job.xml.em

```diff
@@ -72,6 +72,12 @@ parameters = [
   <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
   <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
   <triggers>
+@[if trigger_jobs is not None]@
```

Comment:
The snippet is only needed when there are job names. So the condition should be just `if trigger_jobs`.